src/DotLiquid/Util/ReflectionCache.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace DotLiquid.Util
+{
+    internal class ReflectionCache
```

Comment:
```suggestion
    internal class ReflectionCacheValue
```
This is not the cache itself, but the class for a value within the cache.